mavros_msgs/msg/HilGPS.msg

```diff
@@ -0,0 +1,17 @@
+# HilControls.msg
+#
+# ROS representation of MAVLink HIL_GPS
+# See mavlink message documentation here:
+# https://pixhawk.ethz.ch/mavlink/#HIL_GPS
+
+std_msgs/Header header
+uint8 fix_type
+sensor_msgs/NavSatFix fix
```

Comment:
Well this has a reason: mainly sims like Gazebo like sensor_msgs more than generic types. I can do two things: 1 is to fill the rest of `NavSatFix` message and keep a sustained and complete message. The other is to reverse to a more generic type - probably the `GeoPointInt` is good